cider/exec/template/Execute.cpp

```diff
@@ -334,6 +335,10 @@ StringDictionaryProxy* Executor::getStringDictionaryProxy(
   return row_set_mem_owner->getOrAddStringDictProxy(db_id_, dict_id_in, with_generation);
 }
 
+CiderStringHasher* Executor::getCiderStringHasherHandle() const {
+  return stringHasher_;
+}
+
 StringDictionaryProxy* RowSetMemoryOwner::getOrAddStringDictProxy(
```

Comment:
is it okay to replace `StringDictProxy` with `CiderStringHasher`? For now, I think it won't work for case like `substr(col_str) in ['a', 'b', 'c']`.

Reply:
substring() in case will not work well now. we may need redesign.